x-pack/plugins/observability/common/utils/formatters/formatters.test.ts

```diff
@@ -5,9 +5,45 @@
  * 2.0.
  */
 
-import { asPercent, asDecimalOrInteger } from './formatters';
+import { asDecimal, asInteger, asPercent, asDecimalOrInteger } from './formatters';
 
 describe('formatters', () => {
+  describe('asDecimal', () => {
+    it('displays the not available label when the number is not finite', () => {
+      expect(asDecimal(Infinity)).toBe('N/A');
+      expect(asDecimal(-Infinity)).toBe('N/A');
+      expect(asDecimal(null)).toBe('N/A');
+      expect(asDecimal(undefined)).toBe('N/A');
```

Comment:
This applies to all tests, but could we split the different assertions into different test cases to make them more meaningful?

We could use [`it.each`](https://jestjs.io/docs/api#testeachtablename-fn-timeout) so tests are as easy to write but we get more information in case it fails.